.github/ISSUE_TEMPLATE/icon_request.yml

```
@@ -0,0 +1,94 @@
+name: Icon request
+description: Request a new icon for Simple Icons
+title: "Request: "
+labels: [new icon]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        We won't add non-brand icons or anything related to illegal services. If in
+        doubt, open an issue and we'll have a look. For more details see the
+        [Contributing Guidelines][contributing-guidelines].
+
+
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `new icon` label][new-icon-issues]
+        (including closed issues) before you continue. If you find one for the brand
+        you're requesting then leave a comment on it or add a reaction.
+
+
+        [contributing-guidelines]:
+        https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md
+
+        [new-icon-issues]:
+        https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3A%22new+icon%22+is%3Aopen
+
+  - type: input
+    attributes:
+      label: Brand Name
+      placeholder: "Example: Simple Icons"
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Website
+      description: >-
+        For non-web brands you can add a relevant link or simply put "None" if you don't
+        think there's a website.
```

Minor but I think splitting this into two sentences should make it a bit easier to parse. E.g.:

```suggestion
        For non-web brands you can add a relevant link. You can put "None" if you don't
        think there's a website.
```